improvement: address Bootstrap conflict by renaming our column class (via #4574)

thierrydallacroce opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Both Bootstrap and SwaggerUI rely on a `col` class. When SwaggerUI is added to a site using Bootstrap, the width of some columns is negatively impacted, as seen in screenshots in bugs [4574](https://github.com/swagger-api/swagger-ui/issues/4574) and [5498](https://github.com/swagger-api/swagger-ui/issues/5498).

### Description
<!--- Describe your changes in detail -->
As suggested by @shockey  in their comment [here](https://github.com/swagger-api/swagger-ui/issues/4574#issuecomment-480405337), I replaced the `col` css class to `swui-col`.


### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Use the magic "Fixes #1234" format, so the issues are -->
<!--- automatically closed when this PR is merged. -->
Fixes #4574
Fixes #5498


### How Has This Been Tested?
<!--- Please describe in detail how you manually tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
I ran `npm run test` after my first round of changes, then modified one test which broke since it used `.col` to traverse the DOM, then ran the full test suite again.


### Screenshots (if appropriate):



## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

### My PR contains... 
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

shockey commented:
Actually, after taking a closer look, I don't think we're even using `.col` in our stylesheets. We may be able to remove this entirely...

thierrydallacroce commented:
> Actually, after taking a closer look, I don't think we're even using `.col` in our stylesheets. We may be able to remove this entirely...

No problem, I will remove it then, and update this PR once done.

shockey commented:
@thierrydallacroce sounds good! for the Cypress tests, I think you can just sub out `.swui-col` with `.parameters-col_description`. Ping me if it gives you any trouble, we could look at using a `data-` attribute to target the element instead if Cypress needs a hint to grab what it needs.

thanks!

thierrydallacroce commented:
I removed the column class entirely and used a css attribute selector in Cypress' get() to target either `parameters-col_description` or `parameters-col_name`.

Passed entire test suite twice -- the OAuth2 test timed out once.
Squashed and applied on top of latest upstream/master.

Let me know if that's okay

shockey commented on test/e2e-cypress/tests/bugs/5452.js:
```diff
@@ -8,12 +8,12 @@ describe("#5452: <Select /> crashing in Parameters", function() {
         .get("#operations-default-get_endpoint")
         .click()
-        .get(".parameters > tbody > tr > td[class^='parameters-col_'] > select")
+        .get(".parameters > tbody > tr > .parameters-col_description > select")
```
I think this will suffice, let's see if CI passes 😄

thierrydallacroce commented:
Apologies, I tried replying to your comment but hit Resolve Conversation instead.

You're right, it did suffice, and passed testing :)

shockey commented:
thanks @thierrydallacroce!

Mykolas-Zenitech commented:
Good to see this merged. Which version should I expect the fix to be released in?

webron commented:
@Mykolas-Zenitech we push a release every 1-2 weeks. Since there wasn't one last Friday, it's very likely there will be one this Friday.
